openshift-client/src/main/java/io/fabric8/openshift/client/dsl/internal/BuildConfigOperationsImpl.java

```diff
@@ -229,8 +229,9 @@ public void writeTo(BufferedSink sink) throws IOException {
           try {
             source = Okio.source(inputStream);
             sink.writeAll(source);
-          } finally {
-            Util.closeQuietly(source);
+          } catch (IOException e) {
+            throw KubernetesClientException.launderThrowable("Can't instantiate binary build, due to error reading/writing stream. "
```

Comment:
@iocanel added it to explain to the end user why it might have failed, since the original message wasn't helpful. See https://issues.jboss.org/browse/OSFUSE-492